README.md

```diff
@@ -33,18 +51,42 @@ Run the `runner.py` file located in the root of the project <br />
 This is the preferred method and allows you to include/exclude tags <br />
 a `~` before the tag name excludes it. <br />
 This is how the tests are run on the CI
-#### Example: `python runner.py --tags smoke --tags ~slow`
+<h4> You MUST specify the environment and product
+#### Example: `python runner.py  --product=EPS-FHIR --env=INT --tags smoke --tags ~slow`
 This will run all tests with the tag `@smoke` but skip any tests tagged with `@slow`
 
 ### Method 2:
-If your IDE supports it, you can directly run the .feature files within `/features`
-<br />
-* Note in some IDEs, you will need to set the `BASE_URL` environment within the behave run configuration rather than the machine
+If your IDE supports it, you can directly run the .feature files within `/features` <br />
+Make sure that your behave run configuration includes the `--product=` & `--env=` <B>These are mandatory</B>
 
-### Method 3:
+### Method 3 (Not Recommended):
 Run the tests by running `behave` in a command prompt or terminal window.
 * This will run the tests and print the results to console
 
-### Setting the BASE_URL
-To run the tests from your IDE it is necessary to set the BASE_URL environment variable.
-<br /> If running via GitHub actions this will default to `INT` and currently cannot be changed
+Example:
+```
+behave -D product=EPS-FHIR -D env=INT -f behave_cucumber_formatter:PrettyCucumberJSONFormatter -o reports/cucumber_json.json -f
+allure_behave.formatter:AllureFormatter -o allure-results -f pretty features --no-capture --no-capture-stderr --no-skipped --expand --logging-level=DEBUG --tags eps_fhir
+```
+
+change the `env` variable accordingly to either `INT` or `INTERNAL-DEV`.
```

Comment:
The sentence could start with a capital letter:
Change the `env` variable accordingly to either `INT` or `INTERNAL-DEV`.